README.md

```diff
@@ -61,31 +61,7 @@ What You’ll Need
 Installation
 ------------
 
-### 1. Get a wildcard SSL certificate
```

Comment:
There should be at least one sentence that explains that trusted SSL certificates will be be provided using Let's Encrypt.